### PR TITLE
Unit test 809

### PR DESCRIPTION
**Pull request by eamanu**
Add testcase for PR #809

### CODE REVIEW

**Comment by eamanu** [comment]
Hello, 

I am working to add test case for the PR #809. But if I run the python -m _testcase_ --record this command makes that I login to my repo (eamanu/PyGithub) and I don't have any PR (for example the 31 that is used on the tests) so fail.

What can I do to create testcases?

Thanks!

File: github/PullRequest.py [hunk 1]
```diff
@@ -525,7 +525,7 @@ def get_review_comment(self, id):
             "GET",
             self._parentUrl(self.url) + "/comments/" + str(id)
         )
-        return github.PullRequestComment.PullRequestComment(self._requester, headers, data, completed=True)
+        return github.PullRequestComment.PullRequestComment(self._requester, headers, data, completed=True)
```
Comment (wyardley) [inline github/PullRequest.py]: Looks like you're adding some trailing whitespace here, FWIW

**Comment by wyardley** [comment]
@eamanu I think you'll need to add fake data to https://github.com/PyGithub/PyGithub/tree/master/github/tests/ReplayData

File: github/PullRequest.py (hunk 1 above)
Comment (eamanu) [reply github/PullRequest.py]: Ok, I will fix in the next PR. Thanks

**Comment by eamanu** [comment]
@wyardley  Sorry, I don't understand you. How I add fake data?

**Pull request closed without merging**